src/settings.js

```diff
@@ -16,6 +16,7 @@ const settings = {
   resetOnBlur: true,
   resetOnSelect: true,
   fetchResetOnBlur: true,
+  closeDropdownOnSelect: true,

```

Comment:
Can you just set default value to `false` to keep current behavior?

Reply:
Done!